api/scaling.go

```diff
@@ -40,11 +40,11 @@ func (p *ScalingPolicy) Canonicalize(taskGroupCount int) {
 
 // ScalingRequest is the payload for a generic scaling action
 type ScalingRequest struct {
-	Count  *int64
-	Target map[string]string
-	Reason *string
-	Error  *string
-	Meta   map[string]interface{}
+	Count   *int64
+	Target  map[string]string
+	Message string
+	Error   bool
```

Comment:
stylistic nit, Error being a bool seems unfamiliar, HasError might be a bit verbose :man_shrugging:

Reply:
yeah, we played around with a few names there... it indicates that the autoscaler wants the Nomad operator to know that something is wrong. `HasError` might be better. but `Success/Failed` doesn't feel quite right. 

i wanted to use `ThereIsAProblemInTheAutoscalerOrTheScalingPolicy` but it seemed... not good.

Reply:
`ScalingRequest.HasError` and `ScalingRequest.Failed` both read well, and Failed has a precedent in other events structs